src/main/external-resources/renderers/Sony-PlayStation3.conf

```diff
@@ -39,14 +39,15 @@ Supported = f:mp4        v:mp4|h264               a:aac                m:video/m
 Supported = f:wmv        v:wmv|vc1                a:wma                m:video/x-ms-wmv           n:2
 
 # Supported audio formats:
+Supported = f:aac     a:(?!alac).+   n:2             m:audio/x-m4a
+Supported = f:atrac                  n:2             m:audio/x-oma
+Supported = f:lpcm    a:lpcm         n:6   s:48000   m:audio/l16
```

Comment:
@Sami32 That's already corrected in the merge I'm in the middle of. You should wait with your checks until I'm done with the merge, as this includes all changes done to UMS since I created this PR.